torch/_inductor/config.py

```diff
@@ -531,6 +531,10 @@ class triton:
     # Use cudagraph trees for memory pooling if `cudagraphs` is True
     cudagraph_trees = True
 
+    # Should we skip cudagraphing graphs with dynamic shape inputs
+    # If False, we will re-record a graph for each unique set of shape inputs
+    skip_dynamic_graphs = True
```

Comment:
nit: should we say cudagraphs_skip_dynamic_graphs

Reply:
yea, agreed